R/cox_regression.R

```diff
@@ -186,7 +196,7 @@ tidy.coxreg.univar <- function(x, # nolint
   result
 }
 
-#' @describeIn tidy_coxreg Custom tidy method for a Multi-variable Cox Regression
+#' @describeIn tidy_coxreg Custom tidy method for a multi-variable Cox regression.
```

Comment:
```suggestion
#' @describeIn tidy_coxreg Custom tidy method for a multivariate Cox regression.
```
I think this is better. I talked with Gabe regarding this a couple of weeks ago and he said that in statistics this is the only appropriate form.